Tasks/SonarQubePostTest/PRCA/Orchestrator.ps1

```diff
@@ -21,7 +21,7 @@ function HandleCodeAnalysisReporting
     }	
     else
     {
-        Write-Verbose "The build was not triggered by a Pull Request, not processing code analysis comments"   
+        Write-VstsTaskVerbose "The build was not triggered by a Pull Request, not processing code analysis comments"   
```

Comment:
Writing to verbose works fine too. All pipelines are redirected and converted appropriately by the infrastructure.